### DISCUSSION TITLE
Effects of forest snow cover on deformation

### DISCUSSION THREAD

**Discussion by Congzhangg** [gmtsar/gmtsar#526] [Q&A]
Hello, my research area is a mountain, but there is snow all year round on the top of the mountain and dense jungle on the mountainside, resulting in low interference coherence coefficient, as shown in the figure:（The red box is my research area）
![1667354821352](https://user-images.githubusercontent.com/112800803/199378219-8ddebd78-f3f3-4ec9-a3f7-8dc811598693.png)

In the unwinding process, if the unwinding coefficient is set to be large, such as 0.1, the deformation of the whole mountain will not be monitored. If I had set it lower, like 0.02, I would have gotten a deformable value, but it was far from the GNSS site.
Do you have any good ways to deal with this situation

**Comment by Xiaohua-Eric-Xu** [comment]
1) identify the pixels you want to keep with a stacked coherence grid (similar as coherence masking), create a mask and name it mask_def.grd
2) link this file to your interferogram directories and use `snaphu_interp.csh 0.0001 0` to unwrap while interpolating the bad pixels with near neighbor
3) run sbas.
